app/routes/index.tsx

```diff
@@ -87,25 +97,44 @@ export default function Index() {
 
   return (
     <RootLayout>
-      <div className="max-w-7xl mx-auto pt-12 px-2">
-        <h1 className="text-black-500 font-accent text-3xl lg:text-4xl text-center mb-4">
-          Welcome to Open-Source Hub
+      <div className="banner-bg mx-auto pt-12 md:pt-24 mb-12">
+        <h1 className="text-yellow-300 font-accent text-3xl lg:text-4xl text-center mb-4">
+          Connecting People and Projects
         </h1>
-        <p className="text-black-500 text-center mb-6 mt-2">
-          Connecting 100,000+ potential contributors with maintainers.
+        <p className="text-white text-center px-2 mb-6 mt-2">
+          Your next project is waiting to be discovered. Connect with devs
+          across the
           <br />
-          Helping all onboard better.
+          globe, contribute to open source, and learn something new.
         </p>
```

Comment:
**suggestion:** if I make the page smaller, the `<br />` makes the text wrap in a weird way. I would suggest using a `max-width` on the paragraph to achieve a similar effect. [Tailwind exposes some classes for that](https://tailwindcss.com/docs/max-width). It's okay if the text doesn't wrap _exactly_ like the design.

<img width="524" alt="Screen Shot 2022-05-11 at 10 24 44 AM" src="https://user-images.githubusercontent.com/3411183/167910251-7dccd777-b51b-4d44-8ab7-fa77db596dfc.png">

Reply:
Good catch. It was updated. Feel free to test again.